Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -36,14 +41,16 @@
 static IWLazyClassLookup s_lazyUIFont("UIFont");
 static const float c_defaultSystemFontSize = 15.0f;
 
-static ComPtr<IDWriteFactory> CreateDWriteFactoryInstance() {
+// Private helper for creating a DWriteFactory
+static ComPtr<IDWriteFactory> __CreateDWriteFactoryInstance() {
```

Comment:
seconded

---
In reply to: [78223447](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78223447) [](ancestors = 78223447,78114123)